crates/sargon/src/gateway_api/models/logic/request/transaction/preview/opt_ins_v2.rs

```diff
@@ -0,0 +1,37 @@
+use crate::prelude::*;
+
+impl TransactionPreviewRequestOptInsV2 {
```

Comment:
perhaps `HasSampleValues` ? :) if we all allways add them it is very easy to continue that streak, since new types are built from existing types which all have `HasSampleValues` :)